contrib/verify-binaries/verify.py

```diff
@@ -459,14 +460,17 @@ def verify_binary_hashes(hashes_to_verify: t.List[t.List[str]]) -> t.Tuple[Retur
 
 
 def print_output(
-    args: argparse.Namespace,
-    good_trusted: t.List[SigData],
-    good_untrusted: t.List[SigData],
-    unknown: t.List[SigData],
-    bad: t.List[SigData],
-    files_to_hashes: t.Dict[str, str],
-    missing_files=None,
-    ):
```

Comment:
a6dcf2cb0817143beacf0b9004191a0dd6d2e43c Did you mean to change the indentation here? Maybe do so in the first refactor commit?

Reply:
Thanks fixed in next push